webpack/localization-plugin/src/LocalizationPlugin.ts

```diff
@@ -484,22 +402,22 @@ export class LocalizationPlugin implements Webpack.Plugin {
         !configuration.output ||
         !configuration.output.filename ||
         (typeof configuration.output.filename !== 'string') ||
-        configuration.output.filename.indexOf(LOCALE_FILENAME_PLACEHOLDER) === -1
+        configuration.output.filename.indexOf(Constants.LOCALE_FILENAME_PLACEHOLDER) === -1
       ) {
         errors.push(new Error(
           'The configuration.output.filename property must be provided, must be a string, and must include ' +
-          `the ${LOCALE_FILENAME_PLACEHOLDER} placeholder`
+          `the ${Constants.LOCALE_FILENAME_PLACEHOLDER} placeholder`
         ));
       }
     }
     // END configuration
 
     // START options.filesToIgnore
     { // eslint-disable-line no-lone-blocks
-      this._locJsonFilesToIgnore = new Set<string>();
-      for (const locJsonFilePath of this._options.filesToIgnore || []) {
-        const normalizedLocJsonFilePath: string = path.resolve(configuration.context!, locJsonFilePath);
-        this._locJsonFilesToIgnore.add(normalizedLocJsonFilePath);
+      this._filesToIgnore = new Set<string>();
+      for (const filePath of this._options.filesToIgnore || []) {
```

Comment:
This seems less involved than above where you checked if the file was an absolute path, does this matter here? Will it always be relative to the configuration.context?

Reply:
Actually, `path.resolve` handles if the path is already absolute, so the other absolute check is unnecessary. Thanks for noticing that.